src/styles/style.scss

```diff
@@ -0,0 +1,112 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$blue-color: #00acdc;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $main-text-color;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    margin-top: 16px;
+    justify-content: space-between;
+    display: flex;
+    flex-direction: row;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    line-height: 18px;
+
+    &-text {
+      font-size: 12px;
+      color: $secondary-text-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__buy {
+    margin-top: 16px;
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #ffff;
+    background-color: $blue-color;
+    font-size: 14px;
+    font-weight: 700;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+
+    &:hover {
+      color: $blue-color;
+      background-color: #ffff;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:last-child {
+    background-image: url(../images/star.svg);
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
```

Comment:
Apply the entire styling for stars from the previous task and use SCSS features for this. Stars block should be interactive